### DISCUSSION TITLE
Ide helper solves problems and creates new ones.

### DISCUSSION THREAD

**Discussion by spcbfr** [adalessa/laravel.nvim#56] [General]
Hi, I am using your plugin and  I really like it.

I am facing problems with LSP errors tho. not sure if they can be fixed with this plugin or not but this seems like the most suitable place to share this.

Initially I started getting errors because LSP wasn't reconginzing `Auth::user()` I looked it up online and the fix seemed to install the ide helper package and generate it's helper files.

That fixed the issue but created this whole mess in my routes file (and probably other places too)
![image](https://github.com/adalessa/laravel.nvim/assets/77839865/435be175-e4fb-4fb2-9407-8bbc0585c66d)

It's really frustrating because my neovim setup is almost complete minus a few minor things like this. do you face these kinds of errors and if so how do you handle them?

**Comment by adalessa** [comment]
Hi @spcbfr the plugin does not handle this aspecs of notifications this is on the lsp. Are you using intelephense or phpactor ?
Are you in an update laravel project ? 
To share I use phpactor and I don't have these problems with this.

**Reply by spcbfr** [reply]
I am using phpactor like you recommend on the readme.

I'll checkout your config and see what I can improve on

**Reply by adalessa** [reply]
Does This started after installing ide helper?  If you remove it what is the experience for you ?
